storage/innobase/lock/lock0lock.cc

```diff
@@ -6959,7 +6959,8 @@ DeadlockChecker::check_and_resolve(const lock_t* lock, trx_t* trx)
 
 		print("*** WE ROLL BACK TRANSACTION (2)\n");
 #ifdef WITH_WSREP
-		if (wsrep_on(trx->mysql_thd)) {
+		if (wsrep_on(trx->mysql_thd) &&
+                    wsrep_thd_is_SR(trx->mysql_thd)) {
```

Comment:
In the old InnoDB formatting style, lines are split *before* operators (such as `&&`) and should be indented by TAB (interpreted as 8 spaces) and additional spaces for fine-tuning. Here, the continuation line is indented too deep and the line is split *after* the `&&`, not before it.